boot/core/src/boot/task/built_in.clj

```diff
@@ -103,7 +103,6 @@
             (core/empty-dir! tmp)
             (util/info "Checking out %s...\n" path)
             (pod/unpack-jar (.getPath file) tmp
-              :cache false
```

Comment:
AFAICT, the only places `:cache` is currently used is in:

- `jar-entries`  where it toggled the use of a memoized version of the function or not (https://github.com/boot-clj/boot/blob/master/boot/pod/src/boot/pod.clj#L313)
- `copy-url` where it was used to `setUseCaches` on the URLConnection before creating a stream

Neither of these functions are used by `unpack-jar` anymore.